pivy build broken, which in turn breaks freecad

### Describe the bug
pivy fails to build due to being unable to find some files

### Steps To Reproduce
Steps to reproduce the behavior:
try to build pivy from nixpkgs HEAD

### Notify maintainers
@gebner 

```
~/workspace/nixpkgs $ git rev-parse HEAD
a7cace94228394bee9ee93b80ca00c6f0e36e27d
~/workspace/nixpkgs $ nix-build -A pythonPackages.pivy
this derivation will be built:
  /nix/store/ka6pr42q43lymbnf0fc352jkrv8j65gw-python2.7-pivy-0.6.6.drv
building '/nix/store/ka6pr42q43lymbnf0fc352jkrv8j65gw-python2.7-pivy-0.6.6.drv'...
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh

[snipped a bunch of output]

/nix/store/23m5zb5nqpp0bkgp7fc62rph1cgy64a5-soqt-2020-12-05-unstable/include/Inventor/Qt/SoQtCursor.h:38: Error: Unable to find 'Inventor/SbLinear.h'
SWIG did not generate wrappers successfully! ** Aborting **
error: builder for '/nix/store/ka6pr42q43lymbnf0fc352jkrv8j65gw-python2.7-pivy-0.6.6.drv' failed with exit code 1;
       last 10 log lines:
       > /nix/store/23m5zb5nqpp0bkgp7fc62rph1cgy64a5-soqt-2020-12-05-unstable/include/Inventor/Qt/SoQt.h:72: Error: Unable to find 'Inventor/SoDB.h'
       > /nix/store/23m5zb5nqpp0bkgp7fc62rph1cgy64a5-soqt-2020-12-05-unstable/include/Inventor/Qt/SoQt.h:73: Error: Unable to find 'Inventor/errors/SoDebugError.h'
       > /nix/store/23m5zb5nqpp0bkgp7fc62rph1cgy64a5-soqt-2020-12-05-unstable/include/Inventor/Qt/SoQtGLWidget.h:38: Error: Unable to find 'Inventor/SbBasic.h'
       > /nix/store/23m5zb5nqpp0bkgp7fc62rph1cgy64a5-soqt-2020-12-05-unstable/include/Inventor/Qt/SoQtComponent.h:38: Error: Unable to find 'Inventor/SbLinear.h'
       > /nix/store/23m5zb5nqpp0bkgp7fc62rph1cgy64a5-soqt-2020-12-05-unstable/include/Inventor/Qt/viewers/SoQtViewer.h:56: Error: Unable to find 'Inventor/SbLinear.h'
       > /nix/store/23m5zb5nqpp0bkgp7fc62rph1cgy64a5-soqt-2020-12-05-unstable/include/Inventor/Qt/viewers/SoQtViewer.h:61: Error: Unable to find 'Inventor/events/SoKeyboardEvent.h'
       > /nix/store/23m5zb5nqpp0bkgp7fc62rph1cgy64a5-soqt-2020-12-05-unstable/include/Inventor/Qt/viewers/SoQtExaminerViewer.h:38: Error: Unable to find 'Inventor/SbLinear.h'
       > /nix/store/23m5zb5nqpp0bkgp7fc62rph1cgy64a5-soqt-2020-12-05-unstable/include/Inventor/Qt/widgets/SoQtPopupMenu.h:38: Error: Unable to find 'Inventor/SbBasic.h'
       > /nix/store/23m5zb5nqpp0bkgp7fc62rph1cgy64a5-soqt-2020-12-05-unstable/include/Inventor/Qt/SoQtCursor.h:38: Error: Unable to find 'Inventor/SbLinear.h'
       > SWIG did not generate wrappers successfully! ** Aborting **
       For full logs, run 'nix log /nix/store/ka6pr42q43lymbnf0fc352jkrv8j65gw-python2.7-pivy-0.6.6.drv'.
~/workspace/nixpkgs [1] $
```

--- a/pkgs/development/python-modules/pivy/default.nix
+++ b/pkgs/development/python-modules/pivy/default.nix
@@ -35,7 +35,7 @@ buildPythonPackage rec {
   doCheck = false;
 
   postPatch = ''
-    substituteInPlace CMakeLists.txt --replace \$'{SoQt_INCLUDE_DIRS}' \
+    substituteInPlace distutils_cmake/CMakeLists.txt --replace \$'{SoQt_INCLUDE_DIRS}' \
       \$'{Coin_INCLUDE_DIR}'\;\$'{SoQt_INCLUDE_DIRS}'
   '';
 

